app/views/issues/edit.html.erb

```diff
@@ -1,13 +1,13 @@
 <!-- to support view override using hooks -->
 <% call_hook(:view_issues_edit_view_override) %>
 
-<% if @view_issues_edit_view_override[:assets].present? %>
+<% if @view_issues_edit_view_override && @view_issues_edit_view_override[:assets].present? %>
```

Comment:
may be `@view_issues_edit_view_override&.dig(:assets)` could work as well.

Reply:
Yes that's nicer, I'll change it